packages/ui-new/src/utils/ThemeUtil.ts

```diff
@@ -764,15 +828,9 @@ export const elementStyles = css`
     justify-content: center;
     align-items: center;
     position: relative;
-    transition:
```

Comment:
And why we removing transition value?

Reply:
Same for other removed styles, are we sure they are not required anymore?

Reply:
The design team will create separate transitions for each component, i removed them for now since it'll be changed later anyways.